### PR TITLE
add gelf output plugin documentation

### PR DESCRIPTION
**Pull request by alitoufighi**
Just added a fully descriptive documentation for GELF output plugin. I tried to include everything I saw in implementation. Hope this helps.

### CODE REVIEW

**Comment by alitoufighi** [comment]
Fluent Bit has gelf support from long times ago! But it lacked documentation. (From version 0.12 at least)

File: output/gelf.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+| Gelf_Timestamp_Key | Your log timestamp (_SHOULD be set in GELF_) |timestamp |
+| Gelf_Host_Key | Key which its value is used as the name of the host, source or application that sent this message. (**MUST be set in GELF**) | host |
+| Gelf_Full_Message_Key | Key to use as the long message that can i.e. contain a backtrace. (_Optional in GELF_) | full_message |
+| Gelf_Level_Key     | Key to be used as the log level. Its value must be in [standard syslog levels](https://en.wikipedia.org/wiki/Syslog#Severity_level) (between 0 and 7). (_Optional in GELF_) | level |
+| Packet_Size | If transport protocol is `udp`, you can set the size of packets to be sent. | 1420 |
+| Compress | If transport protocol is `udp`, you can set this if you want your UDP packets to be compressed. | true |
+
+### TLS / SSL
+
+GELF output plugin supports TTL/SSL, for more details about the properties available and general configuration, please refer to the [TLS/SSL](../configuration/tls_ssl.md) section.
```
[the remark this answers is not in the document]
Comment (alitoufighi) [reply output/gelf.md]: Fixed.

**Comment by edsiper** [comment]
thanks.

Looking at the changes it's touching 12 files, please rebase on top and fix the PR to touch only the relevant files of the commit.

**Comment by alitoufighi** [comment]
Sorry @edsiper, I am inexperienced in pull request issues. They're fixed now.

**Comment by edsiper** [comment]
thanks!

**Pull request merged**